src/common/box.css

```diff
@@ -0,0 +1,52 @@
+/* margin padding height & width */
+.px-1 {
+  padding: 0 1.25rem;
+}
+
+.py-1 {
+  padding: 1rem 0;
+}
+
+.p-20 {
+  padding: 20px;
+}
+
+.m-auto {
+  margin: 0 auto;
+}
+
+.ml-auto {
+  margin-left: auto;
+}
+
+.m-1 {
+  margin: 1rem;
+}
+
+.mx-1 {
+  margin: 0 1rem;
+}
+
+.my-1 {
+  margin: 1rem 0;
+}
+
+.m-2 {
+  margin: 2rem;
+}
+
+.mx-2 {
+  margin: 0 2rem;
+}
+
+.my-2 {
+  margin: 2rem 0;
+}
+
+.h-100 {
+  height: 100px;
+}
+
+.h-200 {
+  height: 200px;
```

Comment:
rem can be used here.

Reply:
inline styling was to give a custom value and you get a option of specifying an explicit value